lockdown/tests/test_settings.py

```diff
@@ -24,3 +24,19 @@
 )
 
 ROOT_URLCONF = 'lockdown.tests.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [],
```

Comment:
This was a copy/paste from the default settings:

https://github.com/django/django/blob/1.8/django/conf/project_template/project_name/settings.py#L55-L69